tracer/src/Datadog.Trace/Logging/Internal/Sinks/StackTraceRedactor.cs

```diff
@@ -0,0 +1,434 @@
+﻿// <copyright file="StackTraceRedactor.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
```

Comment:
Maybe put links to the .NET Core `StackTrace.cs` and .NET Framework `StackTrace.cs` just for reference?

Reply:
Unsure if those referencesource links are permanent though

Reply:
Good idea. I've added that to the docs (I did a bit of refactoring to rename this to `ExceptionRedactor` as well